tools/precommit/check_file_properties.py

```diff
@@ -64,6 +64,7 @@
     r"__YUKAWA",
     r"__cplusplus",
     r"HIP_VERSION",
+    r"__NO_IPI_DRIVER",
```

Comment:
Are you sure? The flags has been rename to `__NO_SOCKETS` in https://github.com/cp2k/cp2k/commit/cd366a4022a.

Reply:
It shows up when I run make pretty as this symbol is included in one patch